packages/utils/src/phoneNumbers.ts

```diff
@@ -231,6 +231,34 @@ export function anonymizedPhone(phoneNumber: string) {
   return phoneNumber.slice(0, -4) + 'XXXX'
 }
 
+export function getExampleNumber(
+  regionCode: string,
+  showZero: boolean = true,
+  isInternational: boolean = false
+) {
+  let examplePhone =
+    phoneUtil.getExampleNumber(regionCode) ||
+    phoneUtil.getExampleNumber(getRegionCodeFromCountryCode(regionCode) as string)
+
+  if (!examplePhone) {
+    return
+  }
+
+  const formatedExample = phoneUtil.format(
+    examplePhone,
+    isInternational ? PhoneNumberFormat.INTERNATIONAL : PhoneNumberFormat.NATIONAL
+  )
+
+  if (showZero) {
+    if (isInternational) {
+      return formatedExample.replace(/(^\+[0-9]{1,3} |[0-9])/g, (value, _, i) => (i ? '0' : value))
```

Comment:
Can you explain what this is for?

Reply:
It returns the international example phone number with the next format: `+34 000 00 00 00`, so it keeps the prefix as numbers and the rest as `0`.

Is more clear in the test:
https://github.com/celo-org/celo-monorepo/pull/2061/files#diff-69002191f183e7738d1dcea3c12dd480R264-R282

Reply:
I see. Why the lambda function instead of just a '0'?

Reply:
Because the first value is the prefix, that needs to be skipped in the transformation.